src/app/Fake.DotNet.Testing.Expecto/Expecto.fs

```diff
@@ -0,0 +1,135 @@
+/// Contains tasks to run [Expecto](https://github.com/haf/expecto) unit tests.
+module Fake.DotNet.Testing.Expecto
+
+open Fake.Core
+open Fake.Core.Environment
```

Comment:
`RequireQualifiedAccess` was added to Environment in rc005

Reply:
I think more than happened to ````RequireQualifiedAccess```` Environment. 

I now get this runtime error

````Your environment variables look like they are set manually, but you are missing the default variables. Use the `Process.` helpers to change the 'Environment' field to inherit default values! See https://github.com/fsharp/FAKE/issues/1776#issuecomment-365431982````

Reply:
This was introduced somewhere in the beta phase. This will prevent incorrect usage of the API. Basically you almost always want to inherit the default environment variables, but you replaced them with your own map -> this error occurs.